core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -752,4 +854,8 @@ class TestSuiteWriter {
 
 
     private fun useRestAssured() = config.problemType != EMConfig.ProblemType.RPC
+
+    private fun useWireMock(): Boolean {
```

Comment:
this check should be further extended. if no test case in the whole solution has any WM handling, there is no point to add wiremock in the generated JUnit file